### PR TITLE
Update models in models list

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR solves issue #35

### CODE REVIEW

File: get-started/supported-models.mdx [hunk 1]
```diff
@@ -8,49 +8,50 @@ You will find all the list of supported large langugage models and embedding mod
...
-| llama-3-8b-instruct        | low      |
-| mythomax-l2-13b            | low      |
-| remm-slerp-l2-13b          | low      |
-| yi-34-chat                 | low      |
-| zephyr-7b-beta             | low      |
-| chronos-hermes-13b         | low      |
-| mixtral-8x22b              | low      |
-| dolphin-mixtral-8x7b       | low      |
-| rwkv-5-world-3b            | low      |
+|**alias**|**tier**|
```
Comment (nsosio) [inline get-started/supported-models.mdx:11]: My bad it was just a db export, let's keep it the header of before (| **Model**                  | **Tier** |)

File: get-started/supported-models.mdx [hunk 2]
```diff
@@ -8,49 +8,50 @@ You will find all the list of supported large langugage models and embedding mod
...
+|gpt-3.5-turbo|low|
 
 </div>
 
 <div style={{ width: '48%' }}>
 
 ### Supported Embedding Models 
 
-| **Model**                  | **Tier** |
+| **alias**                  | **tier** |
```
Comment (nsosio) [inline get-started/supported-models.mdx:54]: same here

File: get-started/supported-models.mdx (hunk 1 above)
Comment (Anindyadeep) [reply get-started/supported-models.mdx:11]: Oh no worries, let me fix it

File: get-started/supported-models.mdx (hunk 2 above)
Comment (Anindyadeep) [reply get-started/supported-models.mdx]: Done

**Review by nsosio — APPROVED** [review]

**Pull request merged**